app/AppKernel.php

```diff
@@ -21,6 +21,7 @@ public function registerBundles()
         if (in_array($this->getEnvironment(), ['dev', 'test'], true)) {
             $bundles[] = new Symfony\Bundle\DebugBundle\DebugBundle();
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
+            $bundles[] = new Symfony\Bundle\WebServerBundle\WebServerBundle();
```

Comment:
I would really add it just for the dev env, it does not make sense to register it for the test one, right?